### ISSUE TITLE
favico followup

### ISSUE THREAD

**Issue by christinerogers** [aces/gbc_website#10]
@3design per https://github.com/3design/gbc/issues/36
- please followup on if/when favico can be updated to the GBC logo

**Comment by selineriz** [comment]
@3design GBC logo still needs to be updated or is this issue closed?

**Comment by 3design** [comment]
@selineriz This issue still persists as can be seen [here](https://globalbrainconsortium.org/assets/images/content/GBC2024/GBC-online-program-2024.pdf) 

The issue seems to be that when opening files like [PDF's ](https://globalbrainconsortium.org/assets/images/content/GBC2024/GBC-online-program-2024.pdf) and [images](https://globalbrainconsortium.org/assets/images/content/GBC2024/gbc-meeting-2024a.png) , the favicon for the web browser tab does not show up. On the old repo it defaulted to the favicon that was used for the site I had rigged up on 3design.github.io. Since I moved the website from 3design.github.io repo to a different url, the favicon doesn't show at all. I have a hunch that this is a known issue on github pages with having websites exist on url's that are not the native (username).github.io. 

I am still looking for a solution

**Comment by 3design** [comment]
According to a chatgpt query:

"When you open a PDF or an image file directly from a GitHub Pages site, the browser typically doesn’t render an HTML document (where the <head> section would reference the favicon). Instead, it renders the file in its native viewer (e.g., PDF viewer or image viewer), and the favicon isn’t loaded because those files don’t include any HTML to link to the favicon.

Browsers treat such files differently, so here’s why the favicon doesn’t show:

PDF and Image Files: When you open a direct link to a PDF or an image, the browser bypasses the HTML rendering process and opens the file directly. There’s no mechanism in place to include a favicon since the browser is not processing any HTML that would reference it.

You're correct that when you view a PDF hosted on GitHub Pages (e.g., https://your-site.github.io/file.pdf), the browser may sometimes inherit the favicon from the base site (such as 3design.github.io). This behavior depends on the browser and how it handles linked resources like favicons for files that don't contain HTML, such as PDFs or images.

However, in cases where the favicon doesn’t appear as expected for a direct PDF link, the following factors could be influencing it:

Key Considerations:
Base URL Favicon: Browsers will generally try to fetch the favicon from the base URL (/favicon.ico) of your site, even when rendering non-HTML content. So, as long as the favicon is available at the root of your site, most modern browsers should automatically associate it with files like PDFs or images.

Browser Caching: Sometimes, if a favicon doesn't show up when viewing a PDF, it could be due to browser caching. Try clearing your browser cache, or test in an incognito window.

GitHub Pages Behavior: For non-HTML files, GitHub Pages may not always enforce the use of a favicon, as it doesn't serve custom headers for those types of files. Thus, some browsers might not load the favicon as expected.

In summary, as long as the favicon is in the root directory of your GitHub Pages site (/favicon.ico), browsers should automatically apply it when viewing non-HTML files like PDFs or images. If it doesn’t show up, try clearing your cache or checking in other browsers, since browser behavior can vary."

This site is set up with the favicon being located in the `/assets/images/content` folder. I can try to put the favicon in the `/root`directory and change `src/_includes/components/layouts/main.njk` line 6 so the the `href` points to the root folder. Maybe this will encourage the browser to render the favicon.

**Comment by 3design** [comment]
I did some experimenting on my local version. I have tried putting the favicon in the root directory and changing main.njk. The favicon shows up for web pages but does not show up when opening pdf or img files in the tab.
